<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayControlPanel</class>
 <widget class="QWidget" name="PlayControlPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>646</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>视频壁纸</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icon/VideoWallpaper.ico</normaloff>:/icon/VideoWallpaper.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="_vLayoutTop" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="_hLayoutPlayControl">
     <item>
      <widget class="QLabel" name="_labelThumb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="text">
        <string>缩略图</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_vLayoutPlayControl">
       <item>
        <widget class="QLabel" name="_label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="_hSliderProgress">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedAppearance">
          <bool>false</bool>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
         <property name="tickPosition">
          <enum>QSlider::NoTicks</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_hLayoutControlButton">
         <item>
          <widget class="QPushButton" name="_btnStop">
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btnPrve">
           <property name="text">
            <string>前一个</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btnPlayOrPause">
           <property name="toolTip">
            <string>播放或暂停</string>
           </property>
           <property name="text">
            <string>播放</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/icon/play.ico</normaloff>:/icon/play.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btnNext">
           <property name="text">
            <string>后一个</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="_btnShowPlayList">
           <property name="toolTip">
            <string>展开播放列表</string>
           </property>
           <property name="text">
            <string>列表</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="_btnPlayOrder">
           <property name="text">
            <string>随机</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_vLayoutVolume">
       <item>
        <widget class="QSlider" name="_vSliderVolume">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>60</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_btnMute">
         <property name="text">
          <string>静音</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GalleryWidget" name="_widgetPlayList" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GalleryWidget</class>
   <extends>QWidget</extends>
   <header>gallery/gallerywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
